Project MUSE.js

```diff
@@ -86,100 +86,106 @@ function doWeb(doc, url) {
 }
 
 function scrape(doc, url) {
-	let abstract = ZU.xpathText(doc, '//div[@class="abstract"][1]/p');
-	if (!abstract) abstract = ZU.xpathText(doc, '//div[@class="description"][1]');
-	if (!abstract) abstract = ZU.xpathText(doc, '//div[contains(@class, "card_summary") and contains(@class, "no_border")]');
-	var translator = Zotero.loadTranslator('web');
-	// Embedded Metadata
-	translator.setTranslator('951c027d-74ac-47d4-a107-9c3069ab7b48');
-	translator.setHandler('itemDone', function (obj, item) {
-		if (abstract) {
-			item.abstractNote = abstract.replace(/^\s*Abstract/, "").replace(/show (less|more)$/, "").replace(/,\s*$/, "")
-				.trim();
+	let citationURL = ZU.xpath(doc, '//*[contains(concat( " ", @class, " " ), concat( " ", "view_citation", " " ))]//a');
```

Comment:
why not just 
`ZU.xpathText(doc, '//li[@class="view-citation"]/a/@href')`
? And then you can just plug this right into the ZU.processDocuments function

Reply:
yes. this is much simpler.